src/Serializer/AbstractItemNormalizer.php

```diff
@@ -127,6 +129,91 @@ public function denormalize($data, $class, $format = null, array $context = [])
         return parent::denormalize($data, $class, $format, $context);
     }
 
+    /**
```

Comment:
Please mark it as `@internal` because we'll remove it at some point.

Reply:
Done